solr/core/src/java/org/apache/solr/response/transform/ChildDocTransformerFactory.java

```diff
@@ -22,7 +22,7 @@
 import java.io.UncheckedIOException;
 import org.apache.lucene.search.BooleanClause;
 import org.apache.lucene.search.BooleanQuery;
-import org.apache.lucene.search.DocValuesFieldExistsQuery;
+import org.apache.lucene.search.FieldExistsQuery;
```

Comment:
#2632 (draft) was/is about just removing deprecated `(DocValues,Norms)FieldExistsQuery` use (without Lucene dependency change).

Reply:
no longer applicable: https://github.com/apache/solr/pull/2632#issuecomment-3175763315